[Request][8.16] Update nav references for all "Investigation tools" topics (#6000)
elastic/security-docs: docs/osquery/invest-guide-run-osquery.asciidoc

[[run-live-queries-ig]]
=== Run live queries from an investigation guide

> . Go to the *Rules* page. To access it, find **Detection rules (SIEM)** in the main menu or look for “Detection rules (SIEM)” using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field]. 

Reviewer: ```suggestion
. Go to the *Rules* page. To access it, find **Detection rules (SIEM)** in the main menu or by using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field]. 
```
